gitpod-io/gitpod
# Add editor restriction on organization-level

## Description

- [ ] Rebase after https://github.com/gitpod-io/gitpod/pull/19479 is merged

## Related Issue(s)

Fixes #

## How to test

https://hw-org-editor.preview.gitpod-dev.com/workspaces
- Go to your org settings, update available editors
- available editors in create workspace page should changed after it

## Documentation


#### Preview status

gitpod:summary

## Build Options

<details>
<summary>Build</summary>

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

</details>

<details>
<summary>Installer</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment / Integration Tests</summary>

      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Saves cost. Untick this only if you're really sure you need a non-preemtible machine.
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`. If enabled, `with-preview` and `with-large-vm` will be enabled.
</details>

/hold
